packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -223,9 +220,7 @@ export interface IContextualMenuProps extends IBaseProps<IContextualMenu>, IWith
    * Method to override the render of the individual menu items
    * @default ContextualMenuItem
    */
-  contextualMenuItemAs?:
-    | React.ComponentClass<IContextualMenuItemProps>
-    | React.StatelessComponent<IContextualMenuItemProps>;
+  contextualMenuItemAs?: React.ComponentClass<IContextualMenuItemProps> | React.StatelessComponent<IContextualMenuItemProps>;
```

Comment:
Use `React.ComponentType`

Reply:
I didn't actually change the functionality here, this was just the auto formatting performed by the precommit command. Isn't React.**ComponentClass**<IContextualMenuItemProps> and React.**StatelessComponent**<IContextualMenuItemProps>
already using React.**ComponentType** though?

Reply:
I'm pointing out that this should be consolidated into a single type declaration, for future-compatibility.